observations/obs_converters/GSI2DART/mpisetup.f90

```diff
@@ -32,37 +32,18 @@ module mpisetup
 !$$$
 
 use kinds, only: r_kind, r_single, r_double
-
 implicit none
-
 ! mpi definitions.
 include 'mpif.h'
-
 integer, public :: numproc, nproc
 integer, public :: mpi_status(mpi_status_size)
 integer, public :: mpi_realkind
-integer, public :: my_communicator
 
 contains
 
 subroutine mpi_initialize()
 integer :: ierr
-logical :: already
-
-ierr = -999
-call MPI_Initialized(already, ierr)
-if (ierr /= MPI_SUCCESS) then
-   write(*, *) 'MPI_Initialized returned error code ', ierr
-   call exit(-99)
-endif
-if (.not.already) then
-   call mpi_init(ierr)
-   if (ierr /= MPI_SUCCESS) then
-      write(*, *) 'mpi_init returned error code ', ierr
-      call exit(-99)
-   endif
-endif
-
+call mpi_init(ierr)
```

Comment:
why are you removing code that looks to see if mpi is already initialized?  if this code is used anywhere else, you might be breaking it.